pep_sphinx_extensions/tests/pep_zero_generator/test_parser.py

```diff
@@ -6,6 +6,24 @@
 from pep_sphinx_extensions.pep_zero_generator.author import Author
 from pep_sphinx_extensions.pep_zero_generator.errors import PEPError
 from pep_sphinx_extensions.tests.utils import AUTHORS_OVERRIDES
+from pep_sphinx_extensions.pep_zero_generator.constants import STATUS_ACTIVE
+from pep_sphinx_extensions.pep_zero_generator.constants import STATUS_SUPERSEDED
+from pep_sphinx_extensions.pep_zero_generator.constants import STATUS_WITHDRAWN
+from pep_sphinx_extensions.pep_zero_generator.constants import STATUS_ACCEPTED
+from pep_sphinx_extensions.pep_zero_generator.constants import STATUS_DRAFT
+from pep_sphinx_extensions.pep_zero_generator.constants import STATUS_FINAL
+from pep_sphinx_extensions.pep_zero_generator.constants import STATUS_PROVISIONAL
+from pep_sphinx_extensions.pep_zero_generator.constants import STATUS_DEFERRED
+from pep_sphinx_extensions.pep_zero_generator.constants import STATUS_REJECTED
+from pep_sphinx_extensions.pep_zero_generator.constants import STATUS_VALUES
+from pep_sphinx_extensions.pep_zero_generator.constants import SPECIAL_STATUSES
+from pep_sphinx_extensions.pep_zero_generator.constants import HIDE_STATUS
+from pep_sphinx_extensions.pep_zero_generator.constants import DEAD_STATUSES
+from pep_sphinx_extensions.pep_zero_generator.constants import TYPE_INFO
+from pep_sphinx_extensions.pep_zero_generator.constants import TYPE_PROCESS
+from pep_sphinx_extensions.pep_zero_generator.constants import TYPE_STANDARDS
+from pep_sphinx_extensions.pep_zero_generator.constants import TYPE_VALUES
+from pep_sphinx_extensions.pep_zero_generator.constants import ACTIVE_ALLOWED
```

Comment:
Style nit: in pre-commit, I'd rather use the isort/Black defaults as much as possible and avoid `--force-single-line-imports`. 

I find this big wall of almost-duplicate text less readable.

(Which is probably why I only applied isort to the tests to begin with.)

Reply:
I was thinking the same thing; with nested package layouts it makes diffs really hard to read (which is the _raison d' entree_ to begin with), and its not lexicographically sorted either. If you really want to ensure each import is added/changed atomically, you could tweak the black default isort profile to use one import per line even if the line length is shorter than the max (which is still black compatible, and I use it on several projects, e.g.

```python
from pep_sphinx_extensions.pep_zero_generator.constants import (
    TYPE_PROCESS,
    TYPE_STANDARDS,
)
```